### DISCUSSION TITLE
BT-198 error

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#435] [Q&A] [answered]
We want to unpublish certain fields, for example BT-720 and BT-161. We understand that we have to fill in BT-195, BT-196 and BT-197 in XML but BT-198 should be optional. If we leave it out or don't fill it then we get a schema error because according to this: https://github.com/OP-TED/eForms-SDK/blob/develop/schemas/common/EFORMS-ExtensionAggregateComponents-2.3.xsd
PublicationDate should occur at least once 
````
<xsd:complexType name="FieldsPrivacyType">
		<xsd:sequence>
			<xsd:element ref="efbc:FieldIdentifierCode" minOccurs="1" maxOccurs="unbounded"/>
			<xsd:element ref="cbc:ReasonCode" minOccurs="1" maxOccurs="1"/>
			<xsd:element ref="efbc:ReasonDescription" minOccurs="0" maxOccurs="unbounded"/>
			<xsd:element ref="efbc:PublicationDate" minOccurs="1" maxOccurs="1"/>
		</xsd:sequence>
	</xsd:complexType>
````
So this leads us to 2 questions:
1. Is the XSD schema wrong and PublicationDate shouldn't be mandatory?
2. If it is mandatory to set a publication date, then from what date should we calculate it and how far in the future can it be?

**Comment by rouschr** [comment]
Maybe @YvesJo can say something about this.

**Comment by YvesJo** [comment, marked as answer]
The schema has been designed to satisfy eForms requirements and more specifically help verify the provided eForms rules:

_- "If Procedure Legal Basis (BT-01) value is different from ""Directive 2009/81/EC"", then Unpublished Access Date (BT-198) value must be earlier than Notice Dispatch Date (BT-05) value + 5 years."	(BR-0156-0001)

- "If Procedure Legal Basis (BT-01) value is equal to ""Directive 2009/81/EC"", then Unpublished Access Date (BT-198) value must be earlier than Notice Dispatch Date (BT-05) value + 30 years."	(BR-0156-0002)_
